composer/trainer/trainer.py

```diff
@@ -108,6 +108,112 @@
 __all__ = ["Trainer"]
 
 
+class MissingArgumentException(ValueError):
```

Comment:
Should either move this out to a util file or make it a private class prefixed with `_`

Reply:
Good point. I will make this a private helper function rather than a subclass.

Reply:
Converted to a private helper function.